packages/cli/README.md

```diff
@@ -127,3 +127,82 @@ jobs:
       - run: npm ci
       - run: npx code-pushup autorun --upload.apiKey=${{ secrets.PORTAL_API_KEY }}
 ```
+
+## CLI commands and options
+
+### Global Options
+
+| Option           | Type      | Default                 | Description                                                            |
+| ---------------- | --------- | ----------------------- | ---------------------------------------------------------------------- |
+| **`--progress`** | `boolean` | `true`                  | Show progress bar in stdout.                                           |
+| **`--verbose`**  | `boolean` | `false`                 | When true creates more verbose output. This is helpful when debugging. |
+| **`--config`**   | `string`  | `code-pushup.config.js` | Path to the config file, e.g. code-pushup.config.js                    |
+
+### Common Command Options
+
+| Option                      | Type                 | Default  | Description                                          |
+| --------------------------- | -------------------- | -------- | ---------------------------------------------------- |
+| **`--persist.outputDir`**   | `string`             | n/a      | Directory for the produced reports.                  |
+| **`--persist.filename`**    | `string`             | `report` | Filename for the produced reports without extension. |
```

Comment:
I noticed you are using reports in plural in several places. As the ClI only produces one report at a time, I think a singular "report" should be used.

Reply:
There is only one report, but can be multiple report files (`report.json` and `report.md`).